content/en/docs/instrumentation/cpp/getting-started-new.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Getting Started
+description: Get telemetry for your app in less than 5 minutes!
+cSpell:ignore: rolldice
+weight: 10
+---
+
+This page will show you how to get started with OpenTelemetry in C++.
+
+You will learn how to instrument a simple C++ application automatically,
+such that [traces](/docs/concepts/signals/traces/), [metrics](/docs/concepts/signals/metrics/) and [logs](/docs/concepts/signals/logs/) are emitted to the terminal.
+
+## Prerequisites 
+
+Ensure that you have the following installed locally:
+
+- Git
+- C++ compiler supporting C++ version >= 14
+- Make 
+- CMake version >= 3.1
+
+## Example Application 
+
+The following example uses a basic [Oat++](https://oatpp.io/) application. If you are 
+not using Oat++, that's OK - you can use OpenTelemetry C++ with any other web frameworks as well.
+
+## Dependencies 
+
+**To begin, we will install Oatpp locally using the [source code](https://github.com/oatpp) and `make`, following these general steps:**
+
+1. Download oatpp source code:
+
+Obtain the oatpp source code. You can either download the source code from the oatpp Github repository or clone it using Git.
+
+```bash 
+git clone https://github.com/oatpp/oatpp.git
+```
+
+2. Navigate to the oatpp directory:
+
+Access the oatpp directory in your terminal.
+
+```bash 
+cd oatpp
+```
+
+3. Build oatpp:
+
+Build oatpp using the `make` command. This command will trigger the build process specified in the Makefile included in the oatpp source code.
+
+```bash
+make
+```
```

Comment:
There seems to be something missing, since I get "makefile not found", I guess cmake needs to be run first